drivers/i2c/Kconfig.i2c_sim

```diff
@@ -0,0 +1,11 @@
+# Copyright 2020 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config I2C_SIM
+	bool "I2C simulator"
+	depends on ARCH_POSIX
```

Comment:
From a quick look at the code: Is this really dependent on the posix arch or native_posix? I mean, can this be used in say QEMU or any other platform with spare RAM? (similarly to the EEPROM_SIMULATOR)

Reply:
I do agree here that it looks not native posix specific.

Reply:
Done